%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Medium Length Professional CV
% LaTeX Template
% Version 2.0 (8/5/13)
%
% This template has been downloaded from:
% http://www.LaTeXTemplates.com
%
% Original author:
% Rishi Shah 
%
% Important note:
% This template requires the resume.cls file to be in the same directory as the
% .tex file. The resume.cls file provides the resume style used for structuring the
% document.
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%----------------------------------------------------------------------------------------
%	PACKAGES AND OTHER DOCUMENT CONFIGURATIONS
%----------------------------------------------------------------------------------------

\documentclass{resume} % Use the custom resume.cls style

\usepackage[left=0.75in,top=0.6in,right=0.75in,bottom=0.6in]{geometry} % Document margins

\usepackage{hyperref}
\hypersetup{
    colorlinks=true,
    linkcolor=blue,
    filecolor=magenta,      
    urlcolor=cyan,
}

\urlstyle{same}

\newcommand{\tab}[1]{\hspace{.2667\textwidth}\rlap{#1}}
\newcommand{\itab}[1]{\hspace{0em}\rlap{#1}}
\name{Adam Weiner} % Your name

\address{adamcweiner@gmail.com \\ https://github.com/adamcweiner} % Your phone number and email

\begin{document}

%----------------------------------------------------------------------------------------
%	EDUCATION SECTION
%----------------------------------------------------------------------------------------

\begin{rSection}{Education}

{\bf Weill Cornell Graduate School of Medical Sciences} \hfill {\em July 2019 - June 2024} 
\\ Tri-Institutional Ph.D. Program in Computational Biology \& Medicine (CBM)
% \\ \textbf{Memorial Sloan Kettering Cancer Center}
% \\ Computational Oncology, Department of Epidemiology and Biostatistics
\\ Advised by Dr. Sohrab Shah
\\
\\{\bf University of California, Los Angeles} \hfill {\em Sept 2015 - June 2019} 
\\ B.S. in Bioengineering, technical breadth in Computer Science

\end{rSection}

%----------------------------------------------------------------------------------------
%	Research Experience
%----------------------------------------------------------------------------------------

\begin{rSection}{Research Experience}

{\bf Barbara Engelhardt Lab, Gladstone Institutes} \hfill {\em Jan 2025 - present} \\
Bioinformatics Postdoctoral Fellow
\begin{itemize}
  \item Deep learning models for live-cell imaging and genomic data.
\end{itemize}

{\bf Sohrab Shah Lab, Memorial Sloan Kettering Cancer Center} \hfill {\em Jan 2020 - Jan 2025} \\
PhD Student (Jan 2020 - July 2024), Postdoctoral Researcher (July 2024 - Jan 2025)
\begin{itemize}
  \item Developed hierarchical Bayesian model to infer clone cell cycle distributions and replication timing profiles from unsorted single-cell whole genome sequencing (scWGS) data of genomically unstable cancers.
  \item Developed hierarchical Bayesian model to infer evolutionary timing of whole genome doubling events in ovarian cancer using scWGS.
  \item Graph attention autoencoder for analyzing single-cell resolution spatial transcriptomic, proteomic, and immunofluorescence data of the tumor microenvironment.
  % \item Used hierarchical Bayesian model to account for copy number heterogeneity and patient-specific batch effects in single-cell RNA and ATAC data.
  % \item Used scWGS to study co-segregation patterns of copy number alterations in healthy breast tissue from germline BRCA carriers.
\end{itemize}

{\bf Marcin Imielinski Lab, Weill Cornell} \hfill {\em Sept 2019 - Dec 2019} \\
PhD Rotation Student
\begin{itemize}
  \item Investigated multi-way chromatin contacts in cancer genomes with Pore-C (ONT collaboration).
\end{itemize}

{\bf Olivier Elemento Lab, Weill Cornell} \hfill {\em July 2019 - Sept 2019} \\
PhD Rotation Student
\begin{itemize}
  \item Mined cancer drug combination therapy clinical trial results to build a machine learning pipeline for predicting specific adverse events.
\end{itemize}

{\bf Aaron S. Meyer Lab, UCLA} \hfill {\em Sept 2017 - July 2019} \\
Undergraduate Researcher
\begin{itemize}
  \item Built ODE model for common $\gamma$-chain cytokines to infer immeasurable receptor-ligand reaction rates.
  \item Used tensor factorization to map how native and mutant cytokines preferentially activate specific immune cell populations.
  \item Built lineage tree hidden Markov model that used live-cell imaging recordings to quantify heterogeneity of resistance states that arose after combination cancer therapies.
\end{itemize}

% {\bf Peter Reiher \& Leonard Kleinrock Lab, UCLA} \hfill {\em Sept 2018 - May 2019} \\
% Undergraduate Research Fellow
% \begin{itemize}
%     \item Used hemagglutinin sequencing data to predict flu vaccine efficacy against circulating strains. Awarded \$7,500 and lab space by UCLA Internet Research Initiative (IRI).
% \end{itemize}


% {\bf Thomas Vallim \& Elizabeth Tarling Lab, UCLA} \hfill {\em Summer 2017} \\
% Undergraduate Research Intern
% \begin{itemize}
%   \item Developed ChIP-seq analysis pipeline to quantify the dimerization patterns of Maf family transcription factors which play a role in bile acid and lipid metabolism disorders.
% \end{itemize}

\end{rSection}
%----------------------------------------------------------------------------------------
%	Publications
%----------------------------------------------------------------------------------------

\begin{rSection}{Publications}

\underline{A.C. Weiner}, M.J. Williams, H. Shi, I. Vazquez-Garcia, S. Salehi, N. Rusk, S. Aparicio, S.P. Shah, A. McPherson, ``Inferring replication timing and proliferation dynamics from single-cell DNA sequencing data'', \textit{Nature Communications}. 2024; \textbf{15}: 8512. doi: \url{https://doi.org/10.1038/s41467-024-52544-7}.

A. McPherson, I. Vazquez-Garcia, M. Myers, ..., \underline{A.C. Weiner}, ..., S.P. Shah, ``Ongoing genome doubling promotes evolvability and immune dysregulation in ovarian cancer'', \textit{Nature} (in review). bioRxiv doi: \url{https://doi.org/10.1101/2024.07.11.602772}.

M.J. Williams, M.U.J. Oliphant, V. Au, ..., \underline{A.C. Weiner}, ..., S.P. Shah, J.S. Brugge, S. Aparicio, ``Luminal breast epithelial cells of wildtype and BRCA mutation carriers harbor copy number alterations commonly associated with breast cancer'', \textit{Nature Genetics}. 2024. doi: \url{https://doi.org/10.1038/s41588-024-01988-0}.

M. Kim, A. Gorelick, ..., \underline{A.C. Weiner}, ..., S.P. Shah, E. Reznik, ``Single cell mtDNA dynamics in tumors is driven by ongoing co-regulation of nuclear and mitochondrial genomes'', \textit{Nature Genetics}. 2024; \textbf{56}: 889-899. doi: \url{https://doi.org/10.1038/s41588-024-01724-8}.

H. Shi, M.J. Williams, G. Satas, \underline{A.C. Weiner}, A. McPherson, S.P. Shah, ``Allele-specific transcriptional effects of subclonal copy number alterations enable genotype-phenotype mapping in cancer cells'', \textit{Nature Communications}. 2024; \textbf{15}: 2482. doi: \url{https://doi.org/10.1038/s41467-024-46710-0}.

K.N. Dinh , I. Vazquez-Garcia , A. Chan , R. Malhotra , \underline{A.C. Weiner}, A. McPherson, S. Tavare, ``CINner: modeling and simulation of chromosomal instability in cancer at single-cell resolution''. \textit{PLOS Comp Biol} (in review). bioRxiv doi: \url{https://doi.org/10.1101/2024.04.03.587939}.

T. Funnell, C.H. O'Flanagan, M.J. Williams, ..., \underline{A.C. Weiner}, ... S.P. Shah, S. Aparicio, ``Single-cell genomic variation induced by mutational processes in cancer'', \textit{Nature}. 2022; \textbf{612}: 106–115. doi: \url{https://doi.org/10.1038/s41586-022-05249-0}.

% M.J. Williams, T. Funnell, C.H. O’Flanagan, ... \underline{A.C. Weiner}, ... S. Aparicio, S.P. Shah, ``Evolutionary tracking of cancer haplotypes at single-cell resolution'', \textit{bioRxiv}. 2021. doi: \url{https://doi.org/10.1101/2021.06.04.447031}.

A.M. Farhat$^{\ast}$, \underline{A.C. Weiner}$^{\ast}$, C. Posner, Z.S. Kim, B. Orcutt-Jahns, S.M. Carlson, A.S. Meyer, ``Modeling cell-specific dynamics and regulation of the common gamma chain cytokines'', \textit{Cell Reports}. 2021; \textbf{35}: 109044. doi: \url{https://doi.org/10.1016/j.celrep.2021.109044}.


\end{rSection}

%----------------------------------------------------------------------------------------
%	Presentations, Posters, etc.
%----------------------------------------------------------------------------------------

\begin{rSection}{Presentations, Posters, etc.}

\underline{A.C. Weiner}, M.J. Williams, H. Shi, I. Vazquez-Garcia, S. Salehi, N. Rusk, S.P. Shah, A. McPherson ``Single-cell DNA replication dynamics in genomically unstable cancers'', poster presentation at American Association of Cancer Research (AACR) Annual Meeeting, San Diego, 2024.

\underline{A.C. Weiner}, M.J. Williams, H. Shi, A. McPherson, S.P. Shah, ``Probabilistic modeling of single-cell DNA replication dynamics in genomically unstable cancer cells'', poster presentation at Keystone Symposium for Single Cell biology: From Development to Cancer, Keystone (CO), 2023.

\underline{A.C. Weiner}, M.J. Williams, H. Shi, A. McPherson, S.P. Shah, ``Probabilistic modeling of single-cell DNA replication dynamics in genomically unstable cancer cells'', poster presentation at Weill Cornell's du Vigneaud Research Symposium, New York, 2023.

\underline{A.C. Weiner}, M.J. Williams, H. Shi, A. McPherson, S.P. Shah, ``Probabilistic modeling of single-cell DNA replication dynamics in genomically unstable cancer cells'', poster presentation at American Association of Cancer Research (AACR) Annual Meeeting, Orlando, 2023.

\underline{A.C. Weiner}, M.J. Williams, H. Shi, A. McPherson, S.P. Shah, ``Probabilistic modeling of single-cell DNA replication dynamics in genomically unstable cancer cells'', poster presentation at Mathematical and Computational Methods in Cancer and Biology hosted by Columbia University Irving Institute of Cancer Dynamics, New York, 2023.

\underline{A.C. Weiner}, A. McPherson, S.P. Shah, ``Single-cell replication timing and stress in genomically unstable cancers'', poster presentation at Rockefeller University Anderson Cancer Symposium, New York, 2022.

\underline{A.C. Weiner}, A. McPherson, S.P. Shah, ``Single-cell replication timing and stress in genomically unstable cancers'', poster presentation at Cold Spring Harbor Laboratory 86th Quantitative Biology Symposium: Genome Stability \& Integrity, Cold Spring Harbor, 2022.

\underline{A.C. Weiner}, A. McPherson, S.P. Shah, ``Single-cell replication dynamics in genomically unstable cancers'', oral presentation at American Association of Cancer Research (AACR) Annual Meeeting, New Orleans, 2022.

A.M. Farhat$^{\ast}$, A. Lim$^{\ast}$, N.K. Namiri$^{\ast}$, S. Visagan$^{\ast}$, \underline{A.C. Weiner$^{\ast}$}, A.S. Meyer, ``Lineage tree hidden Markov model quantifies intratumor heterogeneity in cancer therapy'', poster \& presentation at UCLA Bioengineering Symposium, Los Angeles, 2019.

\underline{A.C. Weiner}, A.M. Farhat, A.S. Meyer, ``Building a Reaction Model for Common $\gamma$-chain Cytokines'', poster at Biomedical Engineering Society Annual Meeting, Atlanta, 2018.

\end{rSection}


%----------------------------------------------------------------------------------------
%	Honors and Awards
%----------------------------------------------------------------------------------------

\begin{rSection}{Honors and Awards}


\item \textbf{NIH F31 Predoctoral Fellowship} \hfill {\em 2023 - 2024}
\newline
\textit{National Cancer Institute, F31-CA271673}

\item \textbf{AACR Scholar-in-Training Award} \hfill {\em April 2023}
\newline
\textit{Travel award for AACR Annual Meeting 2023}

\item \textbf{NIH T32 Training Award} \hfill {\em2020 - 2022}
\newline
\textit{Awarded by Tri-I CBM Program Leadership, T32-GM132083}

\item \textbf{UCLA Bioengineering Symposium - Best Poster} \hfill {\em March 2019}
\newline
\textit{Awarded by UCLA Bioengineering and Medicine}

\item \textbf{UCLA Internet Research Initiative Prize Winner} \hfill {\em 2018 - 2019}
\newline
\textit{Awarded by UCLA Computer Science}

\item \textbf{Stanley H. Black Memorial Scholarship} \hfill {\em 2017 - 2019}
\newline
\textit{Awarded by UCLA Engineering and JFLA}

\item \textbf{UCLA Dean’s Honor List (x5)} \hfill {\em 2016 - 2018}

\end{rSection}


%----------------------------------------------------------------------------------------
%	Programming
%----------------------------------------------------------------------------------------

\begin{rSection}{Programming}

\textbf{Languages}: Python, R \\
\textbf{ML Frameworks}: Pyro, PyTorch, scikit-learn, JAX, TensorFlow, TensorLy \\
\textbf{UNIX Clusters}: Slurm, LSF \\
\textbf{Miscellaneous}: Git, Snakemake, Docker, scanpy, AnnData, pandas, numpy, SciPy

\end{rSection}

%----------------------------------------------------------------------------------------
% Extra Curricular
%----------------------------------------------------------------------------------------

\begin{rSection}{Extra-Curricular} \itemsep -3pt
\item Student Representative, CBM Directors Steering Committee \hfill {\em 2022 - 2023}
\item Admissions Committee, MSK Comp Bio Summer Program (CBSP) \hfill {\em 2021 - 2023}
\item Chair, CBM Annual Recruitment \hfill {\em 2021 - 2022}
\item Chair, CBM Research in Progress Series \hfill {\em 2020 - 2021}
\item Organizer, CBM Annual Retreat \hfill {\em 2020 - 2022}
\item Member, Tri-I Start-Up Venture Group \hfill {\em 2020 - 2022}
\item Member, Tri-I Science and Educational Policy Association \hfill {\em 2019 - 2021}
\item Member, New York Academy of Sciences (NYAS) \hfill {\em 2019 - 2022}
\item Mentor, UCLA Engineering (MentorSEAS) \hfill {\em 2016 - 2017}

\end{rSection}

%----------------------------------------------------------------------------------------
% Student supervision
%----------------------------------------------------------------------------------------

\begin{rSection}{Student Supervision} \itemsep -3pt
\item BS project on chromosomal instability (CIN) signatures in HGSOC patients \hfill {\em Summer 2023}
\item BS project to improve scWGS copy number calling using variational inference \hfill {\em Summer 2021}

\end{rSection}


%----------------------------------------------------------------------------------------
%	Areas of expertise
%----------------------------------------------------------------------------------------

\begin{rSection}{Areas of expertise}

\textbf{Biology}: cancer genomics, clonal evolution, DNA copy number, cell cycle, mutational signatures, tumor microenvironment, innate immunity, cytokine signaling \\
\textbf{Statistics}: Bayesian probabilistic graphical models, autoencoders, graph neural networks, ODE models, variational inference, tensor factorization, causality, interpretability, regression, trees and forests, clustering, dimensionality reduction \\
\textbf{Data modalities}: bulk and single-cell DNA-, RNA-, and ATAC-seq, ChIP-seq, chromatin conformation capture (Hi-C), spatial multi-panel immunofluorescence (mpIF), spatial transcriptomics (10X Visium, Visium HD, Xenium), circulating tumor DNA-seq (ctDNA), nanopore, flow cytometry, clinical annotation \\
\textbf{Data sources}: Human patients, mice, cell lines, patient-derived xenografts, public repositories (i.e. TCGA, ENCODE)


\end{rSection}

%----------------------------------------------------------------------------------------
%	Thesis committee
%----------------------------------------------------------------------------------------

\begin{rSection}{Thesis Committee}

Dr. Sohrab Shah (sponsor) \\
Dr. Christina Leslie (chair) \\
Dr. Wesley Tansey \\
Dr. John Maciejowski \\
Dr. Andrew McPherson (non-voting) \\
Dr. Elli Papaemmanuil (non-voting)

\end{rSection}


%----------------------------------------------------------------------------------------
%	Coursework
%----------------------------------------------------------------------------------------

\begin{rSection}{Relevant Coursework}

\item \textbf{Foundations of Graphical Models} \newline
Dr. David Blei, Columbia

\item \textbf{Applied Machine Learning} \newline
Dr. Nathan Kallus, Cornell Tech

\item \textbf{Functional Interpretation of High-Throughput Data} \newline
Dr. Jan Krumsiek, Weill Cornell

\item \textbf{Data Structures and Algorithms in Computational Biology} \newline
Dr. Iman Hajirasouliha, Weill Cornell

\item \textbf{Algorithms in Bioinformatics} \newline
Dr. Eleazar Eskin, UCLA

\item \textbf{Computational Genetics} \newline
Dr. Eran Halperin, UCLA

\item \textbf{Machine Learning and Data-Driven Modeling in Bioengineering} \newline
Dr. Aaron Meyer, UCLA

\item \textbf{Systems Biomodeling} \newline
Dr. Joe DiStefano III, UCLA

% \item \textbf{Biotransport and Bioreaction Processes} \newline
% Dr. Daniel Kamei, UCLA

\end{rSection}


\end{document}
